app/views/idv/shared/_document_capture.html.erb

```diff
@@ -8,17 +8,16 @@
      request,
      [front_image_upload_url, back_image_upload_url],
    )
-   session_id = flow_session[:document_capture_session_uuid]
```

Comment:
Reworked this view to just take `document_capture_session_uuid` instead of a whole `flow_session` (since the hybrid mobile controller doesn't have a `flow_session`).

Reply:
See comment above about deploying this separately.

Reply:
Oh, I was thinking that the controller talks to a view across different requests, but it should all be in the same request, so it should be fine.